pkg/fdsummary.cls

```diff
@@ -53,7 +53,7 @@
 \tl_if_eq:NnT \l_dark_mode_env_var_tl {1} {\enableDarkMode{}}
 \ExplSyntaxOff
 
-\LoadClass[a4paper, fontsize=11pt, accentcolor=3b, class=report, custommargins=false, marginpar=false, pdfa=false]{tudapub}
+\LoadClass[a4paper, fontsize=11pt, accentcolor=3b, class=report, custommargins=false, marginpar=false, pdfa=false, T1]{tudapub}
```

Comment:
Statt inputenc (was nur für PDFTex korrekt funktioniert) zu verwenden, können wir die Eingebaute T1 Option der TU Template verwenden

Reply:
Okay, kannst du das anpassen? :)

Dann merge ich das alles.

Reply:
Das hatte ich schon mit dem [Letzten Commit](https://github.com/fdamken/summaries/pull/4/commits/e78eb9f8a3b2cc9f67b6a42c5c731c8bf233c0b9) behoben ^^
(Die Commit Message hat zwei Zeilen, und Github zeigt iwie nur eine Standartmäßig an)

Reply:
Ah stimmt, das habe ich übersehen. Danke!

Dann merge ich das bald-ish.

Reply:
Man müsste dann vlt nochmal jede Zusammenfassung durch gehen, und schauen ob irgendwelche Grafiken invertiert werden müssen, oder ob noch irgendwo Farben Verwendet werden, die man anpassen sollte fürn Dark Mode (hab das bisher nur bei der CNuVS-Zusammenfassung gemacht)

Komme aber frühestens morgen dazu

(Aber wenn ohne Dark Mode Kompilliert wird, breakt das nichts)

Reply:
Das es ohne nichts kaputt macht ist auf jeden Fall gut. Du kannst zum durchgucken vermutlich einfach die all-Datei in /summaries/summaries/cs nutzen (da liegt auch eine Makefile daneben), dann fällt insgesamt ein PDF raus.